src/pages/graphql/schema/products/interfaces/routable.md

```diff
@@ -183,3 +184,39 @@ The following query returns information about the specified URL key. The query c
   }
 }
 ```
+
+## Routable URL
+
+`RoutableUrl` is the default implementation of RoutableInterface. This type is returned when the URL is not linked to an entity.
+
+### Example
+
+In the following example, an internal URL `support.html` is configured to redirect to an external URL `https://support.example.com/` using URL Rewrite.
```

Comment:
Add a sentence explaining why `type` contains a null value.

Reply:
Good point. I added it in an info box right before the example section. Should I move it down here under Example, please let me know!